nucliadb/nucliadb/ingest/chitchat.py

```diff
@@ -133,7 +133,7 @@ async def close(self):
 JsonObject = Dict[str, JsonValue]
 
 
-def build_member_from_json(member_serial: JsonObject):
+def parse_load_score(member_serial: JsonObject) -> float:
```

Comment:
I think it would be nice to refactor this obscure dict into a pydantic model. I've created a new ticket to do it: https://app.shortcut.com/flaps/story/3993

Reply:
Agree